lib/exirc/client.ex

```diff
@@ -589,6 +603,61 @@ defmodule ExIrc.Client do
     send_event {:topic_changed, channel, topic}, new_state
     {:noreply, new_state}
   end
+
+
+  ## WHOIS
+
+
+  def handle_data(%IrcMessage{cmd: @rpl_whoisuser, args: [_sender, nickname, username, hostname, _, realname]}, state) do
+    user = %{nickname: nickname, username: username, hostname: hostname, realname: realname}
+    {:noreply, %ClientState{state|whois_buffers: Map.put(state.whois_buffers, nickname, user)}}
+  end
+
+  def handle_data(%IrcMessage{cmd: @rpl_whoiscertfp, args: [_sender, nickname, "has client certificate fingerprint "<> fingerprint]}, state) do
```

Comment:
Are we certain this text is standardized? It seems really fragile to match on it like this (though I'm not sure there is a better approach).